### PR TITLE
Fixing broken spring appconfig web management link

### PR DESCRIPTION
**Pull request by conniey**
Fixes broken link that breaks build:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=506079&view=logs&j=ad792ccc-30b2-59a1-5b53-546a79d88828&t=620c66ab-f28e-5b0d-c90a-9b2fc3874d5d

### CODE REVIEW

**Comment by conniey** [comment]
@chenrujun Can you look over this PR?

**Review by chenrujun — APPROVED** [review]
LGTM. Thank you for you effort.

File: sdk/appconfiguration/azure-spring-cloud-feature-management/README.md [hunk 1]
```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
 
 #### Targeting in an Application
 
-An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-samples/feature-management-web-sample) example project.
+An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](../azure-spring-cloud-feature-management-web) example project.
```
Comment (JimSuplizio) [inline sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: /CC @saragluna @conniey 
@chenrujun, why did you approve this when this link is clearly wrong? The new link for the FeatureFlagDemo doesn't even link to a sample, it's linking to the source directory for azure-spring-cloud-feature-management-web. Unless I'm mistaken this should be linking to https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample where the sample actually is.

Comment (chenrujun) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: @JimSuplizio .
You are right.

Originally, I thought it just change the repo from `spring-cloud-azure` to current repo, I didn't found the path error.
Thank you for pointing out.

And IMU, `FeatureFlagDemo` is a Java class name, this class does not exist any more.
So we should write like this:

```
An example web application that uses the targeting feature filter is available in the [example project](../../spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample) .
```

Comment (saragluna) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: @JimSuplizio thanks for pointing this out. Yes, it should be pointing to the new sample project. @chenrujun I think the `FeatureFlagDemo` is the name of the sample project, but if it is confusing we could change that.

Comment (conniey) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: @chenrujun @saragluna It was brought to my attention that the spring team should probably fix this. I had taken a stab at this (incorrectly). The fix can be made inline to this PR or a new one that addresses this/

Comment (chenrujun) [reply sdk/appconfiguration/azure-spring-cloud-feature-management/README.md:238]: I created a new PR: https://github.com/Azure/azure-sdk-for-java/pull/14547/files

**Pull request closed without merging**